<?xml version="1.0" encoding="utf-8"?>

<launch>
  <arg name="pointlaser_config_file"
       default="cfg/pointlaser_params.yaml"
       doc="YAML file holding the parameters."/>
  <arg name="distobox_usb_port"
       default="/dev/serial/by-id/usb-Silicon_Labs_CP2103_USB_to_UART_Bridge_Controller_0001-if00-port0"
       doc="Path of the USB port that the distobox is connected at."/>
  <arg name="move_arm_manually" default="false"/>
  <!-- pointlaser_comm node and pointlaser_calibration node -->
  <include file="$(find cpt_pointlaser_comm_ros)/launch/pointlaser_comm.launch">
    <arg name="distobox_usb_port" value="$(arg distobox_usb_port)" />
  </include>
  <!-- pointlaser_loc node -->
  <node name="pointlaser_loc"
        pkg="cpt_pointlaser_loc_ros"
        type="kinova_localizer_node"
        output="screen"
        respawn="true">
    <rosparam command="load" file="$(find cpt_pointlaser_loc_ros)/$(arg pointlaser_config_file)" />
    <param name="off_model" value="/home/hilteth/181022_286_cla_c_vf_20150815_SE_nonunified.off" />
    <param name="move_arm_manually" value="$(arg move_arm_manually)" />
  </node>
</launch>
